### ISSUE TITLE
Network Search Shift-Click Functionality

### ISSUE THREAD

**Issue by mvandenburgh** [girder/viime#606]
In the Correlation Network graph, metabolites can be added to the search box by holding down shift and clicking their respective nodes on the graph. Currently this functionality isn't indicated to the user in any way; maybe a popup similar to the ones used in the "Method" radio buttons on the same page could work?

**Comment by jeffbaumes** [comment]
Good idea. I think a help icon in one of the corners of the network view would work.
